src/aof.c

```diff
@@ -815,38 +815,43 @@ int openNewIncrAofForAppend(void) {
 #define AOF_REWRITE_LIMITE_THRESHOLD    3
 #define AOF_REWRITE_LIMITE_MAX_MINUTES  60 /* 1 hour */
 int aofRewriteLimited(void) {
-    int limit = 0;
-    static int limit_delay_minutes = 0;
     static time_t next_rewrite_time = 0;
 
     unsigned long incr_aof_num = listLength(server.aof_manifest->incr_aof_list);
-    if (incr_aof_num >= AOF_REWRITE_LIMITE_THRESHOLD) {
+    if (incr_aof_num < AOF_REWRITE_LIMITE_THRESHOLD) {
```

Comment:
```suggestion
    if (incr_aof_num <= AOF_REWRITE_LIMITE_THRESHOLD) {
```
Let's do perform limit when incr_aof_num > AOF_REWRITE_LIMITE_THRESHOLD